manifest.json

```diff
@@ -6,7 +6,7 @@
         "en": "Open source collaboration platform built for developers",
         "fr": "Plateforme de collaboration open source conçue pour les développeurs"
     },
-    "version": "6.1.0~ynh1",
+    "version": "6.1.0~ynh2",
```

Comment:
As far as I understand, this is required only to force an update on people having already the previous `6.1.0~ynh1` version installed.

But in this PR we have only internal technical changes, and nothing that requires an update from the package users.

Should we remove this bump?

Reply:
or we could wait the next version and merge to master the all thing.